src/model/entities/test/part.spec.ts

```diff
@@ -18,6 +18,76 @@ describe('Part', () => {
     })
   })
 
+  describe('putOnAir', () => {
+    afterEach(() => jest.useRealTimers())
+
+    it('sets part to be on air', () => {
+      const testee: Part = new Part({} as PartInterface)
+
+      testee.putOnAir()
+
+      const result: boolean = testee.isOnAir()
+      expect(result).toBe(true)
+    })
```

Comment:
I think this is one of the few times more than one assertions would be nice.
A simple `expect(testee.isOnAir()).toBe(false)` before we call the implementation just to verify that the Part wasn't initialized to be onAir, which would make it possible for `putOnAir` to not put the Part on air without this test noticing.